src/plugins/data/public/ui/filter_bar/filter_editor/value_input_type.tsx

```diff
@@ -147,6 +147,17 @@ class ValueInputTypeUI extends Component<Props> {
     this.props.onChange(boolValue);
   };
 
+  private onNumberChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Are these cases already covered in unit tests?

Reply:
This file does not have any unit tests, added unit test for this file.

Reply:
Thanks for adding the unit tests! Do they cover the change handler you added?

Reply:
We don't test the methods directly, but we still need to exercise the logic - otherwise, how are we validating it works? I'm mostly reacting to the lack of code coverage below.

Reply:
I agree, but how do we test those private method logics? I thought if the function is not exported by the module, it cannot be called by test code outside the module. Could you provide an example to help me? Thanks!